lottie/src/main/java/com/airbnb/lottie/LottieFeatureFlags.java

```diff
@@ -0,0 +1,62 @@
+package com.airbnb.lottie;
+
+import android.annotation.SuppressLint;
+import android.os.Build;
+
+import com.airbnb.lottie.utils.Logger;
+
+import java.util.HashMap;
+import java.util.Objects;
+
+public class LottieFeatureFlags {
+  public enum FeatureFlag {
+    /**
+     * Merge paths currently don't work if the the operand shape is entirely contained within the
+     * first shape. If you need to cut out one shape from another shape, use an even-odd fill type
+     * instead of using merge paths.
+     */
+    MergePath_19(Build.VERSION_CODES.KITKAT);
```

Comment:
I'd like to avoid underscores in the name if possible. Maybe `MergePathsApi19`?

Reply:
Done.